pkg/delete/persistence/delete_persistence.go

```diff
@@ -50,6 +54,22 @@ type DeleteEntry struct {
 
 	// Scope is the parent scope of a config. This field must be set if a classic config is used, and the classic config requires the scope to be set.
 	Scope string `yaml:"scope,omitempty" json:"scope,omitempty" mapstructure:"scope" jsonschema:"description=The scope of the config to be deleted - required for API configs that require a scope"`
+	// CustomValues holds special values that are not general enough to add as a field to a DeleteEntry but are still important for specific APIs
+	CustomValues map[string]string `yaml:",inline"`
+}
+
+func (d DeleteEntry) HashCode() string {
```

Comment:
I've added a map to the fields of `DeleteEntry` struct. (see line 58) Since maps are not Comparable `DeleteEntry` values cannot be used e.g. as keys in a `map` (which is the case on some places). So instead of the `DeleteEntry`value the string returned by `HashCode()`can be used as a map key instead.